arch/arc/Kconfig

```diff
@@ -408,3 +405,5 @@ config CMSIS_V2_THREAD_MAX_STACK_SIZE
 
 config CMSIS_V2_THREAD_DYNAMIC_STACK_SIZE
 	default 2048 if 64BIT
```

Comment:
we should consider to have a `arch/*/Kconfig.defconfig` pattern similar to boards and SoCs.
But that is for another day and orthogonal to HWMv2.